LeGO-LOAM/src/featureAssociation.cpp

```diff
@@ -38,7 +38,9 @@ class FeatureAssociation{
 
 private:
 
-	ros::NodeHandle nh;
+    enum { SIZE_SCANS = N_SCAN * Horizon_SCAN };
```

Comment:
In this context, would it be better to use a scoped enum?

Reply:
It is already scoped inside the class.

From the outside, it must be called FeatureAssociation::SIZE_SCANS

Any anyway the whole software as a bunch of global variables that are not scoped in utils.h ;)